protocol/docker-compose.yml

```diff
@@ -113,13 +113,19 @@ services:
     volumes:
       - ./localnet/dydxprotocol3:/dydxprotocol/chain/.dave/data
   slinky0:
-    image: ghcr.io/skip-mev/slinky-sidecar:v0.3.2
+    image: ghcr.io/skip-mev/slinky-sidecar:v0.4.1
     entrypoint: >
       sh -c "slinky-config --chain dydx --node-http-url http://dydxprotocold0:1317 --oracle-config-path /etc/oracle.json &&
              slinky --oracle-config-path /etc/oracle.json --update-market-config-path /etc/market.json"
     ports:
       - "8080:8080"
-
+      - "8002:8002" # metrics
```

Comment:
why are we changing this port? I know we are no longer using 8002 for our ingested slinky metrics, but lets keep it at 8080, otherwise we'll build a pipeline for 8080 and then have to migrate it to 8002